grpc-sys/Cargo.toml

```diff
@@ -19,33 +19,23 @@ exclude = [
     "grpc/templates/*",
     "grpc/src/android/*",
     "grpc/src/csharp/*",
-    "grpc/src/boringssl/crypto_test_data.cc",
     "grpc/src/node/*",
     "grpc/src/objective-c/*",
     "grpc/src/php/*",
     "grpc/src/python/*",
     "grpc/src/ruby/*",
-    "grpc/vsprojects/*",
     "grpc/test/core/end2end/*",
     "grpc/third_party/zlib/*",
-    "grpc/third_party/abseil-cpp/*",
     "grpc/third_party/benchmark/*",
     "grpc/third_party/bloaty/*",
-    "grpc/third_party/boringssl/fuzz/*",
-    "grpc/third_party/boringssl/crypto/cipher_extra/test/*",
-    "grpc/third_party/boringssl/third_party/android-cmake/*",
-    "grpc/third_party/boringssl/third_party/wycheproof_testvectors",
-    "grpc/third_party/boringssl/third_party/googletest",
-    "!grpc/third_party/boringssl/third_party/googletest/src/gtest-all.cc",
-    "grpc/third_party/boringssl-with-bazel/*",
-    "grpc/third_party/gflags/*",
```

Comment:
Why removing this?

Reply:
These directories do not exist

Reply:
I can still see "gflags" and "googletest" in https://github.com/grpc/grpc/tree/master/third_party.

Reply:
My mistake